extension/source/Home/Onboarding/OnboardingActionPanel.tsx

```diff
@@ -8,11 +8,14 @@ import WorkflowNumbers from './WorkflowNumbers';
 
 const OnboardingActionPanel: FunctionComponent = () => {
   const [params, setParams] = useSearchParams();
-  const [pageIndex, setPageIndex] = useState(1);
+  const [pageIndex, setPageIndex] = useState(0);
 
   useEffect(() => {
-    const currentPage = Number(params.get('p'));
-    setPageIndex(currentPage - 1);
+    const p = params.get('p');
+
+    if (p !== null) {
```

Comment:
Any chance this could come back as `undefined` or an empty string?
```
Number(undefined)  === NaN
Number("") === 0 // !!"" === true, probably ok that we set on this case
```

```suggestion
    if (p) {
```